lib/src/main/java/uk/gov/di/ipv/core/library/domain/ContraIndicator.java

```diff
@@ -8,13 +8,18 @@
 import java.util.List;
 
 @Getter
-@Builder
+@Builder(toBuilder = true)
 @ExcludeFromGeneratedCoverageReport
```

Comment:
We should probably remove the exclusion if there are tests for this class?

Reply:
Added it with a different issuance date but same other fields for testing purposes.